docs/content/getting-started/index.md

```diff
@@ -77,7 +77,7 @@ In addition to starting Radius services in your Kubernetes cluster, this initial
 
 ## 4. Run the app
 
-Use the below command to run the app in your environment, then access the application by opening [http://localhost:3000](http://localhost:3000) in a browser.
+Use the below command to run the app in your environment:
```

Comment:
feels more natural to put the link after the command is run, I was left sort of looking for it